cg/src/domains/management/subjects/subjects.controller.js

```diff
@@ -1,4 +1,15 @@
 const SubjectsService = require('./subjects.service');
+const _ = require('underscore');
+
+function coverString(string) {
+  const firstCharacter = string[0];
+  const lastCharacter = string[string.length - 1];
+  let coveredString = new Array(string.length).fill('*');
```

Comment:
This is mostly a small note but I would like you to fix it so you understand. 

You can make this `coveredString` variable a const, array items can be reassigned with a const